syntax/rspec.vim

```diff
@@ -4,7 +4,7 @@
 "
 "
 
-runtime! syntax/ruby.vim
```

Comment:
Vim will set the `syntax=ruby`, because of the `vim-ruby`. Then others plugins can add more syntaxes to this and use them all. On my local tests I got `syntax = ruby.tomdoc.rspec`. :smiley: